src/pages/EnablePayments/ActivateStep.js

```diff
@@ -21,24 +27,64 @@ const defaultProps = {
     userWallet: {},
 };
 
-const ActivateStep = props => (
-    <ScreenWrapper>
-        <HeaderWithCloseButton
-            title={props.translate('activateStep.headerTitle')}
-            onCloseButtonPress={() => Navigation.dismissModal()}
-            shouldShowBackButton
-            onBackButtonPress={() => Navigation.goBack()}
-        />
-        <View style={[styles.mh5, styles.flex1]}>
-            {props.userWallet.tierName === CONST.WALLET.TIER_NAME.GOLD && (
-                <Text>{props.translate('activateStep.activated')}</Text>
-            )}
-            {props.userWallet.tierName === CONST.WALLET.TIER_NAME.SILVER && (
-                <Text>{props.translate('activateStep.checkBackLater')}</Text>
-            )}
-        </View>
-    </ScreenWrapper>
-);
+class ActivateStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.renderGoldWalletActivationStep = this.renderGoldWalletActivationStep.bind(this);
+    }
+
+    renderGoldWalletActivationStep() {
+        return (
+            <>
+                <View style={[styles.pageWrapper, styles.flex1, styles.flexColumn, styles.alignItemsCenter, styles.justifyContentCenter]}>
+                    <Icon
+                        src={Illustrations.TadaBlue}
+                        height={100}
+                        width={100}
+                        fill={defaultTheme.iconSuccessFill}
+                    />
+                    <View style={[styles.ph5]}>
+                        <Text style={[styles.mt5, styles.h1, styles.textAlignCenter]}>
+                            {this.props.translate('activateStep.activatedTitle')}
+                        </Text>
+                        <Text style={[styles.mt3, styles.textAlignCenter]}>
+                            {this.props.translate('activateStep.activatedMessage')}
+                        </Text>
+                    </View>
+                </View>
+                <FixedFooter>
+                    <Button
+                        text={this.props.translate('common.continue')}
+                        onPress={PaymentMethods.continueSetup}
```

Comment:
Haven't tested but just wanted to confirm if clicking this will lead the user back to the action they were previously taking i.e. paying via the wallet or transferring their wallet balance?

Reply:
This is a huge pain to test on dev for some reason, but yes. It calls [this function](https://github.com/Expensify/App/blob/57d2d97809913cd029baea399e09155ef24dd916/src/components/KYCWall/BaseKYCWall.js#L86-L93), which at the bottom ends up calling the `onSuccessfulKYC()` prop method.

https://user-images.githubusercontent.com/2229301/185936994-0a904103-7edd-499b-b0ec-b102e1c72a60.mov